README.md

```diff
@@ -47,4 +47,4 @@ Add support for calculating factorial.
 ### 6 - Mix
 
 Add support for mixing all sorts of operations.
-"5x1+2-4!" = 17
\ No newline at end of file
+"5x1+2-4!" = 17
```

Comment:
this is the only change. There should be more changes. Maybe you forgot to push?